pkg/bridge/bridge.go

```diff
@@ -146,6 +146,7 @@ func AddVlan(iface string, vlanId int, mtu int) (*netlink.Vlan, error) {
 
 	parentLink, err := netlink.LinkByName(iface)
 	if err != nil {
+		err = fmt.Errorf("failed to find a link by name %s: %v", iface, err)
```

Comment:
You can just return instead of local var declaration/reassignment

```go
return fmt.Errorf("failed to find a link by name %s: %v", iface, err)
```